settings/settings.schema.json

```diff
@@ -21,11 +21,31 @@
              }
           }
       },
-      "additionalProperties": false       
+      "config": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "paths": {
+            "type": "object",
+            "properties": {
+              "get_repo_path": {
```

Comment:
`get_repo_path` and  `buildspec_path`? Those seem awkward.

Reply:
What about repo_path or clone_path or 
For buildspec_path what about search_path or searchpath.

Any suggestions?

Reply:
If it's nested under paths, then saying path again is redundant. If you are sure about having these sections, and you really want path, I'd just do:

```
path
    clonepath
    searchpath
```
or possibly:

```
path
    clones
    search
```

Reply:
maybe i will move it up one level what about 

```
config
  clonepath: 
  searchpath:
```

Reply:
I think "paths" should still be its own section.

Reply:
okay i took your first suggestion